src/pages/overview/pdf-services-api/howtos/pdf-external-storage-sol.md

```diff
@@ -9,17 +9,19 @@ Adobe PDF Services APIs have added support for accessing files from the external
 
 A **Signed URL** temporarily provides access to a resource. Signed URLs contain user/authentication information, allowing users without credentials to access the given resource.
 
-PDF Services APIs supports the signed URLs of input and output files in the request body. Please refer to the **External Section** in Request Body of [Document Generation API](../../../apis/#tag/Document-Generation) for more details.
+PDF Services APIs supports the signed URLs of input and output files in the request body. Please refer to the **External Section** in Request Body of each API for more details.
 
 <InlineAlert slots="text"/>
 
-Accessing files from External Storage is presently restricted to the [Document Generation API](../../../apis/#tag/Document-Generation). Future releases will include the support of external storage across all PDF Services operations.
+Accessing files from External Storage is presently **not** supported in [Extract PDF](../../../apis/#tag/Extract-PDF) and [Split PDF](../../../apis/#tag/Split-PDF) API, as multi-output APIs are not allowed in External Storage Solution.
```

Comment:
remove 'as multi-output APIs are not allowed in External Storage Solution.'

Reply:
reason is not needed to be specified

Reply:
ok. Added auto tag also in this list